actionpack/test/dispatch/routing_test.rb

```diff
@@ -4066,13 +4077,13 @@ def test_to_is_a_symbol
     assert_match(/:to must respond to/, ex.message)
   end
 
-  def test_missing_action_on_hash
+  def test_missing_action_with_to
     ex = assert_raises(ArgumentError) {
       draw do
         get "/foo/bar", to: "foo#"
       end
     }
-    assert_match(/Missing :action/, ex.message)
+    assert_match(/:to must respond to/, ex.message)
```

Comment:
This used to fail in `check_controller_and_action`, whereas now it fails in the checks for the `to:` option, which seems correct to me.